db/model/Link.ts

```diff
@@ -7,15 +7,18 @@ import {
     Relation,
     In,
 } from "typeorm"
-import { OwidGdocLinkJSON } from "@ourworldindata/utils"
+import { OwidGdocLinkJSON, getLinkType, Url } from "@ourworldindata/utils"
 import { Gdoc } from "./Gdoc/Gdoc.js"
+import { formatUrls } from "../../site/formatting.js"
```

Comment:
nit and not related to this PR at all, but I find it a bit odd that it's named in plural when it only ever takes a single URL as input.

Reply:
It does a global replace, so I think the idea is that it could format multiple URLs in a single string.